File: csharpcore-Verify.xunit/GildedRoseFinal/GildedRose.java
package gildedrose;

import java.util.List;

class GildedRose{

	List<Item> items;

	public GildedRose(List<Item> items){
		this.items = items;
	}

	public void dailyItemUpdate(){
		for(Item item : items){
			item.UpdateSellInDays();
			item.updateQuality();
		}
	}
}

File: csharp.xUnit (Stef)/GildedRosePreFinal/GildedRose.cs
using System.Collections.Generic;

namespace GildedRoseKata;

public class GildedRose(IEnumerable<Item> items)
{
    public void Update()
    {
        foreach (var item in items)
        {
            item.UpdateSellInDays();
            item.UpdateQuality();
        }
    }
}